README.md

```diff
@@ -214,8 +214,8 @@ Example:
 
```

Comment:
## 1.
![image](https://user-images.githubusercontent.com/93039359/214265303-1b91a0d2-489f-42c6-86e7-3eb9bfae02d3.png)
## 2.
![image](https://user-images.githubusercontent.com/93039359/214266099-5a9401c6-6955-4ecd-9579-0252c170ecbb.png)
## 3.
![image](https://user-images.githubusercontent.com/93039359/214266235-1600d73a-3337-4e36-9b41-e10a622a494b.png)
## 4.
![image](https://user-images.githubusercontent.com/93039359/214266376-f5b91eb1-c20d-4a5f-bda0-be00ce9acf82.png)
## 5.
![image](https://user-images.githubusercontent.com/93039359/214266458-f8c8e19c-79df-459d-a42a-4d3f00c08ba4.png)
## 6,
![image](https://user-images.githubusercontent.com/93039359/214266551-f7a4698a-4f1c-491c-9450-10b4ed700186.png)
## 7.
![image](https://user-images.githubusercontent.com/93039359/214266704-09e5f00f-ed32-4026-bcb9-b1baaee43106.png)
## 8.
![image](https://user-images.githubusercontent.com/93039359/214266829-d9f39055-e949-4c0b-9007-aaa2aa52b567.png)
## 9.
![image](https://user-images.githubusercontent.com/93039359/214266902-5fcc3213-1b86-442e-9ac5-0017e0179986.png)


- [ ] You have these sections descriptions in your readme file shown in the image above :arrow_up: Kindly remove all these section descriptions to make your readme file descriptive and clean :+1:

Reply:
Done as advised